DatabaseAndReporting/Integration/03_ManageInfluxDBTimeSeries.md

```diff
@@ -0,0 +1,128 @@
+[!!User interface InfluxDB time series](../UserInterface/01b_InfluxDBTimeSeries.md)
+[!!Manage the connection](./04_ManageConnections.md)
+[!!Manage the queries](./01_ManageQueries.md)
+
+# Manage the InfluxDB time series
+
+After a predefined query has been created to retrieve any required data from your databases, this data can be written to InfluxDB at regular time intervals using an InfluxDB time series, which allows you to collect and process a series of data points over time. For detailed information about InfluxDB, see [InfluxDB](https://docs.influxdata.com/influxdb/v2.6/).
+
+Once the data has been written to InfluxDB, it can be easily visualized and monitored using Grafana, see [Get started with Grafana and InfluxDB](https://grafana.com/docs/grafana/latest/getting-started/get-started-grafana-influxdb/).
```

Comment:
```suggestion
Once the data has been written to an InfluxDB, it can be easily visualized and monitored using Grafana, see [Get started with Grafana and InfluxDB](https://grafana.com/docs/grafana/latest/getting-started/get-started-grafana-influxdb/).
```